nrf_security/src/drivers/nrf_oberon/Kconfig

```diff
@@ -63,7 +102,12 @@ config PSA_CRYPTO_DRIVER_ALG_ECDSA_OBERON
 	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_ECDSA && \
 		     !PSA_CRYPTO_DRIVER_ALG_ECDSA_CC3XX
 
-# PSA_CRYPTO_DRIVER_ALG_GCM_OBERON - Currently not supported
+config PSA_CRYPTO_DRIVER_ALG_GCM_OBERON
+	bool
+	prompt "PSA AES GCM support - OBERON" if !PSA_PROMPTLESS
+	default y if !PSA_DEFAULT_OFF
+	depends on PSA_WANT_ALG_GCM && \
+		   !PSA_CRYPTO_DRIVER_ALG_GCM_CC3XX
```

Comment:
Oberon depends on CC3XX here?

Should this be PSA_CRYPTO_DRIVER_ALG_GCM_OBERON?

Reply:
It is supposed to either be CC3XX or Oberon, So Oberon is only used it the CC3XX support is  not used

Reply:
nrf_oberon is only the fallback if CryptoCell is enabled